### ISSUE TITLE
你好

### ISSUE THREAD

**Issue by xiaojianan123** [Acly/krita-ai-tools#27]
According to your method, there are still no two icons. What's the reason? Mine is 5.2.3

**Comment by BarnFindAI** [comment]
It does not appear to work with 5.2.3.   I downgraded to 5.2.2 and the plug-in works again.

**Comment by hansnolte** [comment]
> It does not appear to work with 5.2.3. I downgraded to 5.2.2 and the plug-in works again.
It works for me in 5.2.5.

**Comment by BarnFindAI** [comment]
Did you get it working in 5.2.6?  It doesn't work for me anymore?

**Comment by hansnolte** [comment]
It works flawless in Krita 5.2.6 under WIN 10 for me.

**Comment by BarnFindAI** [comment]
Humm... ok, i must be doing something wrong.  Let me try again.   Thank you for confirming.

**Comment by BarnFindAI** [comment]
I figured it out.  The first time i deleted the "bin, share and lib" folders and replaced them with the new ones.  The second time i just let them override them instead of deleting them first and it works now.   Thanks again for confirming they work.  Cheets

**Comment by hansnolte** [comment]
Nice that it works now.
Bye
